packages/autorest.python/autorest/codegen/models/operation.py

```diff
@@ -204,19 +204,11 @@ def non_default_error_status_codes(self) -> List[Union[str, int]]:
             )
         )
 
-    def _imports_shared(self, async_mode: bool, **kwargs: Any) -> FileImport:
+    def _imports_shared(self, async_mode: bool, **_: Any) -> FileImport:
```

Comment:
why is kwargs _ here?

Reply:
The `kwargs` is an unused parameter after the refactor, and I can't remove it since some sub classes's override are using this parameter.

I reverted this change and instead added a `# pylint: disable=unused-argument`